pom.xml

```diff
@@ -49,6 +49,12 @@
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>fluent-hc</artifactId>
             <version>${httpclient.version}</version>
+            <exclusions>
```

Comment:
This would pull different version of commons-logging than the one we already have in the application server.